rxbinding-support-v4/src/main/java/com/jakewharton/rxbinding/support/v4/view/RxMenuItem.java

```diff
@@ -0,0 +1,183 @@
+package com.jakewharton.rxbinding.support.v4.view;
+
+import android.graphics.drawable.Drawable;
+import android.support.annotation.CheckResult;
+import android.support.annotation.NonNull;
+import android.view.MenuItem;
+
+import com.jakewharton.rxbinding.internal.Functions;
+import com.jakewharton.rxbinding.view.MenuItemActionViewEvent;
+
+import rx.Observable;
+import rx.functions.Action1;
+import rx.functions.Func1;
+
+import static com.jakewharton.rxbinding.internal.Preconditions.checkNotNull;
+
+/**
+ * Static factory methods for creating {@linkplain Observable observables} and {@linkplain Action1
+ * actions} for {@link MenuItem}.
+ */
+public final class RxMenuItem {
```

Comment:
I don't think we need this whole class. There's no `android.support.v4.view.MenuItem` for it to correspond to. If you want a compat implementation then just an `RxMenuItemCompat` with the `actionViewEvents` methods should be all that's here.